library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -317,28 +324,26 @@ private boolean updateLocked() {
 
         List<DownloadBatch> downloadBatches = batchRepository.retrieveBatchesFor(allDownloads);
         for (DownloadBatch downloadBatch : downloadBatches) {
+            if (downloadBatch.isActive()) {
+                isActive = true;
```

Comment:
Do we want to `break` once we've found an active downloadBatch in order not to go through all of them?